python/Ganga/test/GPI/Tasks/TestCoreTasks.py

```diff
@@ -6,24 +6,26 @@
 class TestCoreTasks(GangaUnitTest):
     """A set of tests to cover both ITask and CoreTask plus related objects"""
 
+    argList = ['300', '300', '300']
+
     def setUp(self):
         """Make sure that the Tasks object isn't destroyed between tests"""
         super(TestCoreTasks, self).setUp()
         from Ganga.Utility.Config import setConfigOption
         setConfigOption('TestingFramework', 'AutoCleanup', 'False')
         setConfigOption('Tasks', 'TaskLoopFrequency', 1)
-        self._numTasks = 50
+        self._numTasks = 25
```

Comment:
I'd keep the number of Tasks higher as this test has picked up a few deadlocks in recent weeks and I don't want to risk missing them.